diff --git a/include/ignition/rendering/Storage.hh b/include/ignition/rendering/Storage.hh
index a5e6a22c..6a380b1d 100644
--- a/include/ignition/rendering/Storage.hh
+++ b/include/ignition/rendering/Storage.hh
@@ -294,16 +294,6 @@ namespace ignition
       public: virtual ~StoreWrapper() { }
     };
 
-    template class Store<Scene>;
-    template class Store<Node>;
-    template class Store<Light>;
-    template class Store<Sensor>;
-    template class Store<Visual>;
-    template class Store<Geometry>;
-    template class Store<SubMesh>;
-    template class Map<Material>;
-    template class CompositeStore<Node>;
-
     /// \def SceneStore
     /// \brief Store of Scene
     typedef Store<Scene> SceneStore;


